src/mono/wasm/runtime/rollup.config.js

```diff
@@ -0,0 +1,22 @@
+import { defineConfig } from 'rollup';
+import typescript from '@rollup/plugin-typescript';
+import { terser } from 'rollup-plugin-terser';
+
+export default defineConfig({
```

Comment:
We should definitely put a detailed comment here or in another file explaining what rollup does for us based on this configuration

Reply:
Emcc doesn't know how to load ES6 module, that's why we need the whole rollup.js
We only use it to squash all the ES6 modules typescript generated into single file wrapped as IFFE.
While we are at it, we could also minify the code in Release mode.